test/e2e/test_cluster_configs/everything.json

```diff
@@ -97,14 +98,16 @@
 				"availabilityZones": [
 					"1",
 					"2"
-				]
+				],
+				"ultraSSDEnabled": true
```

Comment:
what about regions that don't support this yet? is it guaranteed that e2e won't run in those regions?

Reply:
The test config interface supports specific regions, so yeah, we'll need to make sure it is set to only include regions that support ultra ssd for cluster configs that are testing ultra ssd (like this one)